cmd/playground/window_state.go

```diff
@@ -0,0 +1,75 @@
+package playground
+
+import (
+	"time"
+
+	"github.com/odpf/optimus/job"
+)
+
+type state struct {
+	windowV1      job.WindowV1
+	windowV2      job.WindowV2
+	scheduledTime time.Time
+}
+
+// increment the component of the time which the cursor is pointed at
+func (s *state) IncrementTime(increaseBy string) {
```

Comment:
Please change the method name, for example, to `IncrementScheduleTimeOn` and the variable name `cursor cursorPointer`. This will make it easier to read that, we increase schedule time based on the cursor position. And if we do that, we will no longer need `getStringValue` method